public/js/utils/source-map-util.js

```diff
@@ -10,7 +10,7 @@ function generatedToOriginalId(generatedId, url) {
 }
 
 function isOriginalId(id) {
-  return id.match(/\/originalSource/);
+  return id && id.match(/\/originalSource/);
```

Comment:
id's should definitely always exist. I'd rather not make things null-compatible; why would be pass in null? If you do `source.id` and it doesn't have an id that's a big problem. (this is also the reason why I don't like lodash. doing this makes bad code be silent)